bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a directly appointed multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The authors recommend reassigning the blockchain-controlled BitAssets’ issuer field from committee-account to cn-vote, which is a multisignature account composed of seven different parties and having numerous proxy followers, and setting cn-vote as the sole feed producer, until cn-vote can delegate this role to accounts of its own appointment. The authors selected cn-vote for this role because cn-vote has demonstrated an active and vested interest in the stability and viability of the BitAsset products. Should cn-vote accept this appointment, the specification of the BitAsset parameters and feed production will become the responsibility of the cn-vote multisignature account.
+
+As a result of this assignment, the block producers will no longer provide feeds for BitAssets, and the new set of feed producers will no longer be affected by direct stakeholder voting.
+
+Because BitAsset management will no longer be affected by stakeholder voting, the vote-weighting calculations can be amended without detriment so that accounts holding BTS collateral positions will count only collateral above the value of their debt at the feed price. Collateral up to a 1.0 collateral-to-debt ratio is under lien to the BitAsset holder, and therefore does not represent a stake in the blockchain, and will no longer be counted as voting stake. Collateral in excess of a 1.0 collateral-to-debt ratio will continue to be counted in the debt holder’s voting stake.
+
+# Specification
+The blockchain software is designed to support assets with multisignature issuer authorities; therefore, the technical changes required for this proposal are straightforward. The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, the committee-controlled BitAsset objects' issuer fields will be updated to specify the cn-vote account (ID 1.2.1148184). The existing price feeds will be cleared, and the cn-vote account will be specified as the feed producer. After the hardfork, the cn-vote account will be able to assign other feed producers at its discretion.
+
+**Disable the witness_fed_asset and committee_fed_asset Flags**
+
+At the hardfork date, the `witness_fed_asset` and `committee_fed_asset` flags will be unset from the BitAssets, and will be disabled from use in future assets.
```

Comment:
By "and will be disabled from use in future assets." you mean assets issued by cn-vote, not the whole bitshares network right? Because witnesses should still be free to feed 'private' smartcoins on the bitshares network in the future.

Reply:
Every smartcoin can define a list of accounts that should be price feeders. There is nothing that stops anyone from listing witness accounts there.
What we propose here is to remove the flag that connects the witness *job* to the pricefeeder job.

Reply:
> 
> 
> Every smartcoin can define a list of accounts that should be price feeders. There is nothing that stops anyone from listing witness accounts there.
> What we propose here is to remove the flag that connects the witness _job_ to the pricefeeder job.

This is not a correct solution, and you didn't give a clear governance programmes.

Reply:
>  What we propose here is to remove the flag that connects the witness job to the pricefeeder job.
Given the committee never provides price feeds themselves, why offer committee as an option either?
An advantage of using the witness fed setting is that it's a permanent setting, so if I got hit by a bus and all witnesses switched for new witnesses in 10 years time it would still be fully operational.

Reply:
Resolved through elected entities.